cmd/dave/main.go

```diff
@@ -76,6 +76,12 @@ func wrapRecovery(handler http.Handler) http.Handler {
 			}
 		}()
 
+		if len(config.Cors.Origin) > 0 {
+			w.Header().Set("Access-Control-Allow-Origin", config.Cors.Origin)
+			w.Header().Set("Access-Control-Allow-Headers", "*")
```

Comment:
Here you could insert the other configurable config.Cors properties if introduced before.

Reply:
> I would also like to see the new additional properties in examples/config-sample.yaml. Commented out of course. ;)

Done.

Reply:
Thanks :)